src/plugins/kibana_react/public/code_editor/editor.scss

```diff
@@ -27,4 +27,21 @@
       display: none;
     }
   }
+
+  &__controls {
+    top: 2px;
```

Comment:
These `px` values should be avoided and can be replaced with their equivalent [EUI sizing variables](https://eui.elastic.co/#/theming/sizing#scale) like:

```
$euiSizeXS / 2; // 2px
$euiSize; // 16px
$euiSizeL; // 24px
```